packages/snjs/lib/services/Features/FeaturesService.ts

```diff
@@ -63,6 +64,7 @@ export class SNFeaturesService
   private deinited = false
   private roles: RoleName[] = []
   private features: FeatureDescription[] = []
+  private experimentalFeatures = GetExperimentalFeatures()
```

Comment:
Hmm it seems like some of my comments got deleted? Maybe the git-sync deletes comments because it force pushes? Can you check your email for some comments on this.